src/xcode/ENA/ENA/Source/AppDelegate & Globals/CWAHibernationProvider.swift

```diff
@@ -28,7 +28,11 @@ class CWAHibernationProvider: RequiresAppDependencies {
 		if isUITesting {
 			return LaunchArguments.endOfLife.isHibernationStateEnabled.boolValue
 		}
-		return secureStore.hibernationComparisonDate >= hibernationStartDate
+		Log.debug("current hibernationStartDate \(secureStore.hibernationStartDate)")
+		return secureStore.hibernationStartDate >= Date()
+		#elseif !RELEASE
+		Log.debug("current hibernationStartDate \(secureStore.hibernationStartDate)")
+		return secureStore.hibernationStartDate >= Date()
```

Comment:
I don't understand the intension of that code. 
Example: 
Today = 09.03.2023
secureStore.hibernationStartDate = 10.03.2023

`return secureStore.hibernationStartDate >= Date() // hibernation would be true`

Hibernation would be true every time the set date in Dev Menu is later than today.
Hibernation would be false every time the set date in Dev Menu is earlier than today

Reply:
yes that is what the testers are expecting

Reply:
Then we should have this requirements in a ticket description. Nobody could comprehend the changes.